README.md

```diff
@@ -15,8 +15,9 @@
 # 📣 What does this dbt package do?
```

Comment:
* Readme updated
* Passthrough column section removed
* Wording changed to a recommendation
* Sadly none that I could find. All the extractor documentation seems to live inside of SAP transactions but otherwise can only be found from third party sources via Google. I'll ask Edwin about this.